docs/model-types.mdx

```diff
@@ -0,0 +1,467 @@
+---
+title: ML Models in MindsDB
+---
+
+In this section, we go through the ML model types available in MindsDB. These are regression models, classification models, and time series models.
+
+## What is an ML Model?
+
+A machine learning (ML) model is a program trained using the available data in order to learn how to recognize patterns and behaviors and make decisions that lead to predicting future data.
+
+You input data into an ML model. It processes this input data, searching for patterns and correlations. After that, the ML model returns output data defined based on the input data.
+
+### Features
+
+Features define the columns or variables that an ML model uses as input data to search for patterns and predict the target variable.
+
+### Target
+
+A target is a column or variable that an ML model predicts based on the information fetched out of the features.
+
+### Training Dataset
+
+The training dataset is used during the training phase of an ML model. It contains both feature variables and a target variable. As its name indicates, it is used to train an ML model.
+
+The ML model takes the entire training dataset as input. It learns the patterns and relationships between feature variables and correlates them with the target variable values.
```

Comment:
and ~correlates them with the~ target ~variable~ values.

Reply:
Changed:

> The ML model takes the entire training dataset as input. It learns the patterns and relationships between feature variables and target values.